tests/integration/cocoon/test_cocoon_failures.py

```diff
@@ -365,7 +385,7 @@ def test_load_from_data_frame_failure(
             ]
         ]
     )
-    def test_load_from_data_frame_failure(
+    def test_load_from_data_frame_failure2(
```

Comment:
why was this renamed to `2`? can we have a more descriptive name

Reply:
@rizwansaeed Turns out that if two methods have the exact same names in the same class, then only one of them is returned by the inspect method. That's the reason for name change